### ISSUE TITLE
Category changes don't get reflected immediately.

### ISSUE THREAD

**Issue by THElegend5** [tachiyomiorg/tachiyomi#8222]
### Steps to reproduce

1. Change category of something from the library screen.

### Expected behavior

Category changes get reflected immediately.

### Actual behavior

Category changes don't get reflected immediately and require a tab change to reflect.

### Tachiyomi version

Preview r5062

### Android version

Android 11

### Device

Redmi Note 9 Pro Max

### Other details

The library reloads again every time you change a tab and go into a manga & come back which is probably fixed in #8221 , this also causes the library to not crash on only removing multiple from a category more details in comments of #7997.

https://user-images.githubusercontent.com/63903377/196086600-a317680e-2a16-4f0a-ab4b-b402bd1a4454.mp4

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.13.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by THElegend5** [comment]
The download count and read and unread count changes don't get reflected either after reading a chapter.

**Comment by arkon** [comment]
Seems fine after #8240 at least.
